index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="pl">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <form class="add" action="">
+      <input type="text" />
+      <button>Dodaj</button>
+    </form>
+    <input class="search" type="text" onfocus='this.value=""' value="Szukaj" />
+    <h1>Liczba zadań: <span>0</span></h1>
+    <ul></ul>
+    <script src="main.js"></script>
```

Comment:
Czemu tutaj dodane a nie w head? Jaka jest różnica?

Reply:
Jeżeli dodam skrypt w head to zostanie on wczytany przed body, w tym momencie dla skryptu reszta strony poprostu nie istnieje. Wynika to z DOM. Dlatego uważam że dobrą praktyką jest dodawanie skryptu na samym końcu kodu jak wszystkie elementy html zostaną wczytane.